### DISCUSSION TITLE
Install Compiler Explorer on local machine (Windows 11 23H2)

### DISCUSSION THREAD

**Discussion by ghost** [compiler-explorer/compiler-explorer#6491] [Q&A] [answered]
Does anyone know how I can install this Compiler Explorer repository on my local Windows 11 machine?

**Comment by partouf** [comment, marked as answer]
We have some documentation https://github.com/compiler-explorer/compiler-explorer/blob/main/docs/WindowsNative.md
It might be slightly out of date, but it can probably get you going
